stealthy-require breaks things

currently comes up with `request-promise` usage.

Things break unless you manually require `https` / `http` before requiring `request-promise`

e.g.:

```
const beeline = require('honeycomb-beeline')({});

require('https'); // if I comment this out, I get an 'api.traceActive is not a function' error
const request = require('request-promise-native');

const trace = beeline.startTrace({
  task: 'http request',
});

beeline.startAsyncSpan({ name: "request"}, span => {
  return request({
    method: 'GET',
    url: 'https://www.google.com',
  }).then(() => {
    beeline.finishSpan(span);
  });
}).then(() => {
  beeline.finishTrace(trace);
});
```

I ran into this same `api.traceActive is not a function` error.  In my case it was with running a Node.js 10.X app managed by PM2 that connects to SQS (stack trace below). 

It turns out that PM2 was the issue.  I was running an older version of PM2, 2.X, and upgrading it to 4.X fixed this for me.

The solution above of manually adding the `require('https')` worked for me as well, however upgrading PM2 will be my long term fix in this case.

```
[ERROR] - { NetworkingError: api.traceActive is not a function
     at Object.request (/var/www/node_modules/honeycomb-beeline/lib/instrumentation/https.js:25:16)
     at features.constructor.handleRequest (/var/www/node_modules/aws-sdk/lib/http/node.js:42:23)
     at executeSend (/var/www/node_modules/aws-sdk/lib/event_listeners.js:304:29)
     at Request.SEND (/var/www/node_modules/aws-sdk/lib/event_listeners.js:318:9)
     at Request.callListeners (/var/www/node_modules/aws-sdk/lib/sequential_executor.js:101:18)
     at Request.emit (/var/www/node_modules/aws-sdk/lib/sequential_executor.js:77:10)
     at Request.emit (/var/www/node_modules/aws-sdk/lib/request.js:683:14)
     at Request.transition (/var/www/node_modules/aws-sdk/lib/request.js:22:10)
     at AcceptorStateMachine.runTo (/var/www/node_modules/aws-sdk/lib/state_machine.js:14:12)
     at /var/www/node_modules/aws-sdk/lib/state_machine.js:26:10
     at Request.<anonymous> (/var/www/node_modules/aws-sdk/lib/request.js:38:9)
     at Request.<anonymous> (/var/www/node_modules/aws-sdk/lib/request.js:685:12)
     at Request.callListeners (/var/www/node_modules/aws-sdk/lib/sequential_executor.js:115:18)
     at callNextListener (/var/www/node_modules/aws-sdk/lib/sequential_executor.js:95:12)
     at /var/www/node_modules/aws-sdk/lib/event_listeners.js:220:9
     at finish (/var/www/node_modules/aws-sdk/lib/config.js:320:7)
     at /var/www/node_modules/aws-sdk/lib/config.js:338:9
     at Credentials.get (/var/www/node_modules/aws-sdk/lib/credentials.js:126:7)
     at getAsyncCredentials (/var/www/node_modules/aws-sdk/lib/config.js:332:24)
     at Config.getCredentials (/var/www/node_modules/aws-sdk/lib/config.js:352:9)
     at Request.SIGN (/var/www/node_modules/aws-sdk/lib/event_listeners.js:192:22)
     at Request.callListeners (/var/www/node_modules/aws-sdk/lib/sequential_executor.js:101:18)
message: 'api.traceActive is not a function',
code: 'NetworkingError',
... }
```

--- a/README.md
+++ b/README.md
@@ -17,6 +17,8 @@ This package makes it easy to instrument your Express/NodeJS application to send
 
 - Using a bundler (esbuild, webpack, etc.) or ESM with the Beeline is unsupported. You may be able to use the Beeline in those cases, but auto-instrumentations will likely not work.
 
+- An error like `'api.traceActive is not a function' error` may occur for some auto-instrumentation and can be resolved by manually requiring `https` before requiring other libraries such as `request-promise`.
+
 ## Contributions
 
 Features, bug fixes and other changes to `beeline-nodejs` are gladly accepted. Please

